### PR TITLE
Bug/landing page font

### PR DESCRIPTION
**Pull request by jimjamesjimathy**
I resized the font's so that they would not overlap and become muddy when screen is resized. I also implemented flexbox and media queries so that the font would remain readable and in its correct place when viewed on alternate devices and screen sizes.

Fixes # (font overlapping and images bleeding over)

## Loom Video

https://www.loom.com/share/76b5140f98054644850ca6d8b8bc212b

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Landing/RenderLandingPage.js [hunk 1]
```diff
@@ -80,12 +80,12 @@ function RenderLandingPage(props) {
           <div className="landing-second-container-text">
             <Title
               className="landing-second-container-text landing-second-container-title-one"
-              level={3}
+              level={2}
             >
-              Give Back To Community
+              Give Back To The Community
```
Comment (dstrazzeri) [inline src/components/pages/Landing/RenderLandingPage.js:85]: Great catch!

**Review by dstrazzeri — APPROVED** [review]
I can't hear your video, but that might be a user error on my end. You did explain this to us in the standup though. Regardless, It looks great! So much better.

**Comment by dstrazzeri** [comment]
I went ahead and resolved the merge conflict on this PR, just had to remove the "parentDashboard" import from index.js

**Comment by rjgeng** [comment]
Hi, Always perfect job, This is our pair programming, I have double check here, A great job.

**Review by AndrewGary — APPROVED** [review]
Changes look great, great styling.

**Pull request merged**